### PR TITLE
display character names in drop down

### PR DESCRIPTION
**Pull request by isose**
Character names are displayed in drop down after submitting a valid account name.

### CODE REVIEW

**Comment by isose** [comment]
[Display character names in drop down](https://trello.com/c/B9kp5LSh/28-display-character-names-in-drop-down)

File: client/src/LandingPage.js [hunk 1]
```diff
@@ -66,9 +76,18 @@ function LandingPage() {
                   value={character}
                   onChange={(e) => setCharacter(e.target.value)}
                 >
-                  <option className="option-placeholder" value="" disabled>
+                  <option hidden value="">
                     Select your character
                   </option>
+                  {account.characters.map(
+                    ({ name, level, class: clazz, league }) => {
```
Comment (theninjawoohoo) [inline client/src/LandingPage.js:83]: i just wondering why we are using clazz as the temp variable for class,  educate me if the naming is okay.

Comment (isose) [reply client/src/LandingPage.js:83]: class is a reserved keyword, so it can't be used as a variable name. Common conventions use clazz or cls instead.

Comment (theninjawoohoo) [reply client/src/LandingPage.js:83]: sounds good, thanks for letting me know.

**Review by theninjawoohoo — APPROVED** [review]

**Review by AndyLiuCodes — APPROVED** [review]
LGTM

**Pull request merged**